src/library/Uno.Material/Styles/Controls/NavigationView.xaml

```diff
@@ -0,0 +1,1273 @@
+														<!-- Set negative top margin to make the flyout align exactly with the button -->
+														<Setter Property="Margin"
+																Value="0,-4,0,0" />
+													</Style>
+												</Flyout.FlyoutPresenterStyle>
+												<NavigationViewList x:Name="TopNavMenuItemsOverflowHost"
+																	ItemTemplate="{TemplateBinding MenuItemTemplate}"
+																	ItemTemplateSelector="{TemplateBinding MenuItemTemplateSelector}"
+																	ItemContainerStyle="{TemplateBinding MenuItemContainerStyle}"
+																	ItemContainerStyleSelector="{TemplateBinding MenuItemContainerStyleSelector}"
+																	ScrollViewer.HorizontalScrollBarVisibility="Hidden"
+																	ScrollViewer.VerticalScrollBarVisibility="Hidden"
+																	SingleSelectionFollowsFocus="False"
+																	IsItemClickEnabled="True">
+													<ListView.ItemContainerTransitions>
+														<TransitionCollection />
+													</ListView.ItemContainerTransitions>
+												</NavigationViewList>
+											</Flyout>
+										</Button.Flyout>
+									</Button>
+
+									<ContentControl x:Name="PaneCustomContentOnTopPane"
+													IsTabStop="False"
+													VerticalContentAlignment="Stretch"
+													HorizontalContentAlignment="Stretch"
+													Grid.Column="5" />
+
+									<Grid x:Name="TopPaneAutoSuggestArea"
+										  Height="{ThemeResource NavigationViewTopPaneHeight}"
+										  Grid.Column="6">
+
+										<ContentControl x:Name="TopPaneAutoSuggestBoxPresenter"
+														Margin="12,0,12,0"
+														MinWidth="48"
+														IsTabStop="False"
+														HorizontalContentAlignment="Stretch"
+														VerticalContentAlignment="Center" />
+									</Grid>
+
+									<ContentControl x:Name="PaneFooterOnTopPane"
+													IsTabStop="False"
+													VerticalContentAlignment="Stretch"
+													HorizontalContentAlignment="Stretch"
+													Grid.Column="7" />
+
+									<NavigationViewItem x:Name="SettingsTopNavPaneItem"
+														Style="{ThemeResource NavigationViewSettingsItemStyleWhenOnTopPane}"
+														Grid.Column="8"
+														Icon="Setting" />
+
+								</Grid>
+
+								<Border x:Name="TopNavContentOverlayAreaGrid"
+										Child="{TemplateBinding ContentOverlay}" />
+							</StackPanel>
+
+							<SplitView x:Name="RootSplitView"
+									   Background="{TemplateBinding Background}"
+									   CompactPaneLength="{TemplateBinding CompactPaneLength}"
+									   DisplayMode="Inline"
+									   IsPaneOpen="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=IsPaneOpen, Mode=TwoWay}"
+									   IsTabStop="False"
+									   OpenPaneLength="{TemplateBinding OpenPaneLength}"
+									   PaneBackground="{StaticResource NavDrawersBackgroundColor}"
+									   Grid.Row="1">
+
+								<!-- Known Issue ElevatedView creates abnormal and unwanted behavior inside splitview pane -->
```

Comment:
if we didnt use ElevatedView I wouldn't mention it here, potentially in a issue but we should only document in code what is in the code :D

Reply:
whoops, didnt see this one, will remove in next pr